h-util-ui/client/src/components/TaskList/TaskListener.ts

```diff
@@ -0,0 +1,29 @@
+import { SpawnedTask } from '@shared/common.types';
+import { getIpcRenderer } from '@utils/helpers';
+import throttle from 'lodash/throttle';
+
+class TaskListener {
```

Comment:
May transition this out so the component doesn't do listening, but for now, the issue was on the back end using the same reference to the ID.